bin/i18n/test/translation_status/test_translation_service.rb

```diff
@@ -6,50 +6,54 @@ class TranslationServiceTest < Minitest::Test
   @@translation_service = TranslationService.new
 
   def test_translated_given_english_hello_should_return_true
-    assert @@translation_service.translated?('en-US', 'hello')
+    assert @@translation_service.translated?('en-US', 'hello', '')
   end
 
   def test_translated_given_lowercase_english_locale_should_return_true
-    assert @@translation_service.translated?('en-us', 'hello')
+    assert @@translation_service.translated?('en-us', 'hello', '')
   end
 
   def test_translated_given_german_hello_should_return_true
-    assert @@translation_service.translated?('de-DE', 'hello')
+    assert @@translation_service.translated?('de-DE', 'hello', '')
   end
 
   def test_translated_given_es_hello_should_return_true
-    assert @@translation_service.translated?('es-ES', 'hello')
+    assert @@translation_service.translated?('es-ES', 'hello', '')
   end
 
   def test_translated_given_pt_hello_should_return_true
-    assert @@translation_service.translated?('pt-BR', 'hello')
+    assert @@translation_service.translated?('pt-BR', 'hello', '')
   end
 
   def test_translated_given_a_unknown_string_key_should_return_false
-    refute @@translation_service.translated?('en-US', 'xxxxxxxxxxxxxxxxxxxxx_does_not_exist')
+    refute @@translation_service.translated?('en-US', 'xxxxxxxxxxxxxxxxxxxxx_does_not_exist', '')
   end
 
   def test_translated_given_a_unknown_locale_should_return_false
-    refute @@translation_service.translated?('xx-XX', 'hello')
+    refute @@translation_service.translated?('xx-XX', 'hello', '')
   end
 
   def test_translated_given_dashboard_string_should_return_true
-    assert @@translation_service.translated?('es-MX', :unplugged_activity)
+    assert @@translation_service.translated?('es-MX', :unplugged_activity, '')
   end
 
   def test_translated_given_pegasus_string_should_return_true
-    assert @@translation_service.translated?('es-MX', :anybody_can_learn)
+    assert @@translation_service.translated?('es-MX', :anybody_can_learn, '')
   end
 
   def test_translated_given_blockly_mooc_string_should_return_true
-    assert @@translation_service.translated?('de-DE', 'turtle.loopVariable')
+    assert @@translation_service.translated?('de-DE', 'turtle.loopVariable', '')
   end
 
   def test_translated_given_blockly_core_string_should_return_true
-    assert @@translation_service.translated?('de-DE', 'core.INLINE_INPUTS')
+    assert @@translation_service.translated?('de-DE', 'core.INLINE_INPUTS', '')
```

Comment:
You could set empty string as the default value for `scope` to make the code a bit cleaner.

Reply:
Yes, good catch!